Gallery/Panels/NCL_panel_4.py

```diff
@@ -68,13 +75,18 @@ def plot_labelled_filled_contours(data, ax=None):
         tick labels.
     """
 
-    # Import an NCL colormap, truncating it by using geocat.viz.util convenience function
-    newcmp = gv.truncate_colormap(cmaps.gui_default, minval=0.03, maxval=0.9)
+    # Import an NCL colormap, truncate it, and save the colormap
+
+    # Import the colormap
+    cmap = mpl.cm.get_cmap('OrRd', 13)
+    # Create a linear segmented colormap using the colormap
```

Comment:
Line 82 through 84 can be removed (or just double check that it isn't accidentally being commented out if it is required to address a different topic)